docs/src/lang/sets.md

```diff
@@ -106,12 +106,12 @@ If this is not the case, the type checker flags an error.
                             \* Model checking error in TLC, type error in Apalache
 ```
 
-**Example in Python:** TLA+ sets are immutable, so we are using `frozenset`:
+**Example in Python:**
 
 ```python
-  frozenset({1, 2, 3})
-  frozenset({frozenset({1, 2}), frozenset({2, 3})})
-  frozenset({False, 1})
+{1, 2, 3}
+{frozenset({1, 2}), frozenset({2, 3})}
+{False, 1}
```

Comment:
To be consistent, I guess these should also be runnable, like on lines 373-379?

Reply:
I think doctests only make sense when the output is of interest, as in lines 373-379 where some expressions get evaluated. Here it would simply be
```python
>>> {1, 2, 3}
{1, 2, 3}
>>> {frozenset({2, 3}), frozenset({1, 2})}
{frozenset({2, 3}), frozenset({1, 2})}
>>> {False, 1}
{False, 1}
```
Seems strange doesn't it?

Reply:
I prefer the consistency at a price of a bit of redundancy, but I'm ok with leaving it this way if you prefer. I can see your point.

Reply:
Sure, I think it doesn't make much of a difference.